kubernetes/synapse.yml.erb

```diff
@@ -36,11 +41,20 @@ spec:
           env:
           - name: "SYNAPSE_CONFIG_PATH"
             value: "/etc/matrix/homeserver.yaml"
+        - name: matrix-appservice-irc
```

Comment:
this will create a second container in the same pod, which is almost definitely not what we want. From https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/:
> *Pods that run multiple containers that need to work together.* A Pod might encapsulate an application composed of multiple co-located containers that are tightly coupled and need to share resources. These co-located containers might form a single cohesive unit of service–one container serving files from a shared volume to the public, while a separate “sidecar” container refreshes or updates those files. The Pod wraps these containers and storage resources together as a single manageable entity.

We should probably just use multiple deployments. Take a look at how ocfweb does this.

Reply:
+1, these should definitely run in separate deployments